docs/components/board/upboard.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Configure an upboard board"
+linkTitle: "upboard"
+weight: 70
+type: "docs"
+description: "Configure an upboard board."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components"]
+# SMEs: Susmita
+---
+
+Configure an `upboard` board to integrate an Intel-based board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) into your robot.
+
+## Set up requirements
```

Comment:
Hmm that'd be a big reorg. Most of the boards have a "requirements" box that is pretty attention-grabbing right now (though not in keeping with the style guide, since it's a unique title and it's a use of caution when it should be green or blue). I think it makes a certain amount of sense to have those setup guides in a separate place because:
- You can install viam-server on them before/without ever configuring a board, so it's really a separate process from configuration
- Some of those setup instructions are long and wouldn't fit nicely on the config page

Options that seem reasonable to me:
- Move these setup instructions to the setup guide, and follow the current format of the other boards with this one (i.e. yellow req's box). Be sure to mention the Setup tab at the end of such instructions.
- Since these instructions are quite brief compared to other setup guides, leave them on this page, and put a little "Important" box at the top saying you have to do this
- Leave as-is, because frankly, these instructions are right here in front of peoples' faces and if they made it to this page, they will likely see them. This page isn't terribly long.

I'm kinda worried that folks won't even arrive at this page at all--should this board be part of the carousel? And if you decide not to put it in the setup guide, do you think it'd make sense to add a link to it from the setup guide? Or is this an obscure board such that we don't really care?